Revit_Core_Engine/Query/DuctSectionProfile.cs

```diff
@@ -54,23 +54,14 @@ public static BH.oM.MEP.SectionProperties.SectionProfile DuctSectionProfile(this
             // Insulation thickness
             double insulationThickness = revitDuct.LookupParameterDouble(BuiltInParameter.RBS_INSULATION_THICKNESS);
```

Comment:
Should be changed to "RBS_REFERENCE_INSULATION_THICKNESS" as the current param value returns zero. This switch has worked on my tests to return accurate values for ducts and all profiles. More testing is required for pipes etc. 

Likewise, above for the lining "RBS_REFERENCE_LINING_THICKNESS"

Reply:
This is odd, the built in parameters are mapped to their corresponding string parameter names, but for some reason it doesn't return anything.
![image](https://user-images.githubusercontent.com/28534672/93354983-7ea63b80-f835-11ea-97ac-f8fed68e63cc.png)

![image](https://user-images.githubusercontent.com/28534672/93355213-b8774200-f835-11ea-9bc8-229fa099e644.png)

Reply:
Agreed, I was seeing the same thing.

Reply:
Fixed